### ISSUE TITLE
automatically add a condition when creating a new event with alt-create ?

### ISSUE THREAD

**Issue by bltzr** [ossia/score#789]
this used to work in v1:
![v1-alt](https://user-images.githubusercontent.com/113982/41675305-a16a6e1e-74c2-11e8-9863-137069215c25.gif)

in v2, we have to manually add the condition afterwards:
![v2-alt](https://user-images.githubusercontent.com/113982/41675313-a7713dc4-74c2-11e8-8470-bd95f9b5c737.gif)

but maybe that's better this way, what do you think @jln- ?
IMHO, there's no reason to create another event if there's no condition, but I might be overseeing aspects of the problem ?

**Comment by bltzr** [comment]
though, in v1 I'm suspecting this created the problem described in https://github.com/OSSIA/score/issues/648 which won't be missed :-)

**Comment by jln-** [comment]
> IMHO, there's no reason to create another event if there's no condition

Yes, I was surprised not to see a condition added but was not sure my memory was correct. Glad it was.

But I agree, at first thought, I need see a reason either not to add it.
